internal/acctest/checks.go

```diff
@@ -110,6 +111,77 @@ func PreCheckRoot(t *testing.T) {
 	}
 }
 
+// PreCheckServerExposed skips the test if the server is not exposed on the localhost
+// over port 8443. This is required for remote provider tests.
+func PreCheckLocalServerHTTPS(t *testing.T) {
+	nc, err := exec.LookPath("nc")
+	if err != nil {
+		t.Skip(`Skipping remote provider test. Command "netcat" not found`)
+	}
+
+	err = exec.Command(nc, "-z", "127.0.0.1", "8443").Run()
+	if err != nil {
+		t.Skip(`Skipping remote provider test. LXD is not available on "https://127.0.0.1:8443"`)
+	}
```

Comment:
Please do that TCP open check in go.

Reply:
Yeah really good point. Thanks, don't know what I was thinking.. 👯